code/modules/reagents/chemistry/reagents/water.dm

```diff
@@ -304,6 +304,13 @@
 			if(5 to 12)
 				to_chat(M, "<span class = 'danger'>You feel an intense burning inside of you!</span>")
 				M.adjustFireLoss(1)
+				M.Stuttering(1)
+				M.Jitter(20)
+				if(prob(20)) M.emote("scream")
+				if(M.mind.vampire.bloodusable > 10)
+					M.mind.vampire.bloodusable = M.mind.vampire.bloodusable - 10
+				else
+					M.mind.vampire.bloodusable = 0
```

Comment:
The vampire's blood should deplete before they take actual damage, otherwise they're going to be killed when sec tries to "peacefully" restrain them.
And when the blood level hits 0, it should flush all of the holy water from the vamp's system, to avoid them from getting killed by the "remainder" of the holy water, unless they acquire a second dose at which point they get rekt as normal